packages/open-next/src/core/routing/middleware.ts

```diff
@@ -162,7 +162,15 @@ export async function handleMiddleware(
       const rewriteUrlObject = new URL(rewriteUrl);
       newUrl = rewriteUrlObject.pathname;
       //reset qs
-      middlewareQueryString = {};
```

Comment:
Don't leave an empty line between comment and the code that the comment explains.